watertap/property_models/activated_sludge/modified_asm2d_reactions.py

```diff
@@ -64,20 +57,14 @@ class ModifiedASM2dReactionParameterData(ReactionParameterBlock):
     CONFIG.declare(
         "decay_switch",
         ConfigValue(
-            default=DecaySwitch.on,
-            domain=In(DecaySwitch),
+            default=True,
+            domain=Bool,
             description="Switching function for decay",
-            doc="""
-           Switching function for handling decay in reaction rate expressions.
-
-           **default** - `DecaySwitch.on``
-
-       .. csv-table::
-           :header: "Configuration Options", "Description"
-
-           "``DecaySwitch.on``", "The decay of heterotrophs and autotrophs is dependent on the electron acceptor present"
-           "``DecaySwitch.off``", "The decay of heterotrophs and autotrophs does not change"
-       """,
+            doc="""Switching function for handling decay in reaction rate expressions,
+**default** - True.
+**Valid values:** {
+**True** - the decay of heterotrophs and autotrophs is dependent on the electron acceptor present,
+**False** - the decay of heterotrophs and autotrophs does not change""",
```

Comment:
@MarcusHolly Does this need to be updated on the technical ref ?

Reply:
It is updated automatically
![image](https://github.com/watertap-org/watertap/assets/96305519/659391e3-77dc-4671-8830-c708bc480b6e)

Reply:
I suspected this but didn't check if any manually written description conflicted with this.

Reply:
The manual descriptions of decay are all still accurate I believe.